src/util/plotting.jl

```diff
@@ -0,0 +1,52 @@
+using RecipesBase
+
+@recipe f(gp::AbstractGP, x::Array) = gp(x)
+@recipe f(gp::AbstractGP, x::AbstractRange) = gp(x)
+@recipe f(gp::AbstractGP, xmin::Number, xmax::Number) = (gp, range(xmin, xmax, length=1000))
```

Comment:
```suggestion
@recipe f(gp::AbstractGP, xmin::Real, xmax::Real) = (gp, range(xmin, xmax, length=1000))
```
We probably don't want to allow complex numbers here :)